data/ui/LedDialog.ui

```diff
@@ -7,11 +7,11 @@
     <property name="step_increment">1</property>
     <property name="page_increment">10</property>
   </object>
-  <object class="GtkAdjustment" id="adjustment_effect_rate">
-    <property name="lower">100</property>
-    <property name="upper">20000</property>
-    <property name="step_increment">50</property>
-    <property name="page_increment">50</property>
+  <object class="GtkAdjustment" id="adjustment_effect_duration">
+    <property name="lower">0</property>
+    <property name="upper">10000</property>
+    <property name="step_increment">500</property>
+    <property name="page_increment">500</property>
```

Comment:
half-second increments seem a bit high. Maybe make it at 100 for step and 500 for page?